bundles/AdminBundle/public/js/pimcore/document/document.js

```diff
@@ -115,7 +117,7 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
             }
 
             Ext.Ajax.request({
-                url: Routing.getBaseUrl() + "/admin/" + this.getType() + '/save?task=' + task,
+                url: Routing.generate(this.getSaveRoute()),
```

Comment:
`?task` param is missing here, leading to [problems](https://github.com/pimcore/pimcore/blob/68e86786778387b23aa20de74a7812d84a8688b6/bundles/AdminBundle/src/Controller/Admin/Document/DocumentControllerBase.php#L381)